diesel/src/query_builder/insert_statement/mod.rs

```diff
@@ -240,6 +240,28 @@ where
     }
 }
 
+#[cfg(feature = "sqlite")]
+impl<T, U, Op> ExecuteDsl<SqliteConnection>
```

Comment:
This impl bothers me more than the rest of the duplication. I'd like to mull on it for a bit.